platforms/jvm/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/dependencies/TestSuitesGroovyDSLDependenciesIntegrationTest.groovy

```diff
@@ -756,151 +520,60 @@ class TestSuitesGroovyDSLDependenciesIntegrationTest extends AbstractIntegration
         succeeds 'checkConfiguration'
     }
 
-    def "can add dependency with actions on suite using a #desc"() {
+    // region multiple GAV strings
+    def "can add multiple of GAV dependencies to #suiteDesc - at the top level (varargs)"() {
```

Comment:
```suggestion
    def "can add multiple GAV dependencies to #suiteDesc in single method call - at the top level (varargs)"() {
```

Reply:
Should I be changing existing test names here? This really hasn't been changed at all, GitHub diff is just misrepresenting it.

Reply:
I think it's fine to improve a method name to make it a little clearer, even if not directly changed.